### ISSUE TITLE
Remove an useless dev-dep

### ISSUE THREAD

**Issue by HerryYT** [nearform/fast-jwt#300]
Seems like there's a dependency that it's using an useless development dependecy
```
C:\Users\xxx\Documents\GitHub\Mitmarine>pnpm i -D fast-jwt           
 WARN  deprecated @types/minipass@3.3.5: This is a stub types definition. minipass provides its own type definitions, so you do not need this installed.
 ```

**Comment by simoneb** [comment]
Solved in #301

**Comment by HerryYT** [comment]
Thank you very much! @simoneb can you bump minor & publish?
